Exercise_1.py

```diff
@@ -1,19 +1,32 @@
 class myStack:
      def __init__(self):
-         
+         self.items= []
```

Comment:
This implementation works but uses inbuilt data structure. Please implement your own stack using top pointer